tests/testthat.R

```diff
@@ -1,4 +1,11 @@
 library(testthat)
 library(chromote)
 
+on_cran <- !interactive() && isTRUE(as.logical(Sys.getenv("NOT_CRAN", "false")))
+
+if (!on_cran && !identical(Sys.getenv("CHROMOTE_CHROME"), "")) {
```

Comment:
```suggestion
if (!on_cran && identical(Sys.getenv("CHROMOTE_CHROME"), "")) {
```

This was intended to be _if `CHROMOTE_CHROME` is unset_: if not on CRAN and if we don't already have a CHROMOTE_CHROME envvar, then get and use latest stable Chrome.